clients/web/src/views/layout/HeaderUserView.js

```diff
@@ -13,10 +13,9 @@ girder.views.LayoutHeaderUserView = girder.View.extend({
         },
 
         'click a.g-logout': function () {
-            girder.restRequest({
-                path: 'user/authentication',
-                type: 'DELETE'
-            }).done(_.bind(function () {
+            girder.logout();
+
+            girder.events.once('g:logout.success', _.bind(function () {
```

Comment:
I think this will make it so `g:login` gets triggered twice, once here and once by `girder.logout`. I think it might make sense to remove this binding altogether and simply have `girder.login` and `girder.logout` update the value of `girder.currentUser` internally.